src/main/java/frc/robot/subsystems/gripper/GripperInputs.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.units.*;
+import org.littletonrobotics.junction.AutoLog;
+
+@AutoLog
+public class GripperInputs {
+    public MutableMeasure<Angle> currentAngle;
+    public MutableMeasure<Angle> angleSetpoint;
+    public double rollerPowerSetpoint;
+    public MutableMeasure<Voltage> angleMotorVoltage;
+    public MutableMeasure<Voltage> rollerMotorVoltage;
+    public boolean hasNote;
+    public MutableMeasure<Velocity<Velocity<Distance>>> acceleration;
+    public boolean atSetpoint;
```

Comment:
Like @katzuv said, this shouldn't be an input.

Reply:
a554e0b5cf343ecb191494bf7cbf3e4f4c1c3394

Reply:
d66ef4afb7db3ba53292f6119a1ced03f66dd314